app/database/models/usersModel.js

```diff
@@ -0,0 +1,31 @@
+const db = require('../dbConfig');
+
+/**
+ * get user by id
+ * @param id
+ * @returns array []
+ */
+async function getById(id){
+    const rows = await db('users')
+        .where({id})
+        .select('*');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8408698)
```suggestion
        .select("*");
```